test/functional/Java8andUp/src_110_up/org/openj9/test/annotation/ContainsRuntimeAnnotationTest.java

```diff
@@ -280,6 +284,16 @@ public void test_overridden_methods() throws Exception {
 		Assert.assertTrue(annotationFound, "I.method3 doesn't have MyMethodAnnotation annotation");
 	}
 
+	@Test
+	public void test_dont_inline() throws Exception {
```

Comment:
can you also add the src_80 version

Reply:
I thought you couldn't import ForceInline and DontInline in java8

Reply:
ah right, I forgot